autoload/utils/common.vim

```diff
@@ -22,10 +22,11 @@ function! utils#common#executeCommands(cmds, open_result) abort
     " add all fields
     let l:commands = []
     for l:cmd in a:cmds
-        let l:dict_cmd = {'cmd': l:cmd['cmd']}
-        let l:dict_cmd['cwd'] = get( l:cmd, 'cwd', getcwd()  )
-        let l:dict_cmd['errFormat'] = get( l:cmd, 'errFormat', ''  )
-        let l:commands += [l:dict_cmd]
+        call add(l:commands, #{
```

Comment:
Note that this is valid `vimscript` (`:h #{}`), but I guess the linter doesn't like it.